### ISSUE TITLE
Cannot use the `RESTORE` command.

### ISSUE THREAD

**Issue by JoverZhang** [laixintao/iredis#467]
Cannot use the `RESTORE` command.

To reproduce:

```shell
iredis  1.13.1 (Python 3.11.3)
redis-server  7.2.0
Home:   https://iredis.io
Issues: https://iredis.io/issues
127.0.0.1:6379> set a 1
OK
127.0.0.1:6379> get a
"1"
127.0.0.1:6379> dump a
"\x00\xc0\x01\x0b\x00O\xa7Z\xc5,\x9e\xf8u"
127.0.0.1:6379> restore b 0 "\x00\xc0\x01\x0b\x00O\xa7Z\xc5,\x9e\xf8u"
(error) ERROR DUMP payload version or checksum are wrong
```

**Comment by laixintao** [comment]
Thanks for reporting, I will take a look

**Comment by laixintao** [comment]
Fixed in latest version, will release soon

**Comment by laixintao** [comment]
released in 1.13.2
